src/pages/term/Term.tsx

```diff
@@ -35,12 +49,31 @@ export default function Term(props: TermProps) {
 
     setTerm(term);
 
-    fetch(`/terms/${term.markdownPath}.markdown`)
+    fetch(`/terms/${term.customMarkdownPath}.md`)
       .then(res => res.text())
       .then(text => setMarkdown(text))
       // eslint-disable-next-line no-console
       .catch(error => console.error(error));
   }, [props.name]);
```

Comment:
Should be in a `useCallback` hook. It'll calculate the Markdown elements everytime, even if the `markdown` prop hasn't changed.
```suggestion
  const markdownContent = useCallback(() => {
    markdown ? Markdown(markdown) : null;
  }, [markdown]);
  
```

Reply:
I used the useMemo() hook instead. The useCallback hook prevents unnecessary rendering of functions, rather than values.

Reply:
Great optimization